# Modules avx

This document lists software modules available with AVX support.  Information includes module name, type, versions, description, homepage URL, and keywords.


| Module          | Type    | Versions                                      | Description                                                                                                                                                                                                                         | Homepage                                                                        | Keyword(s) |
|-----------------|---------|-------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|-------------|
| abaqus          | phys    | 2021                                          | Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA.                                                                                                                      | [http://www.simulia.com/products/abaqus_fea.html](http://www.simulia.com/products/abaqus_fea.html) | phys        |
| abyss           | bio     | 2.2.5                                          | Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler                                                                                                                                                         | [https://www.bcgsc.ca/platform/bioinfo/software/abyss](https://www.bcgsc.ca/platform/bioinfo/software/abyss) | bio         |
| actc            | -       | 1.1                                           | ACTC converts independent triangles into triangle strips or fans.                                                                                                                                                                     | [https://sourceforge.net/projects/actc](https://sourceforge.net/projects/actc)     |             |
| admixture       | bio     | 1.3.0                                          | ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much more rapidly using a fast numerical optimization algorithm. | [http://software.genetics.ucla.edu/admixture](http://software.genetics.ucla.edu/admixture) | bio         |
| advisor         | tools   | 2020.3                                         | Vectorization Optimization and Thread Prototyping - Vectorize and thread code or performance dies - Easy workflow + data + tips = faster code faster - Prioritize, Prototype and Predict performance gain                            | [https://software.intel.com/intel-advisor-xe](https://software.intel.com/intel-advisor-xe) | tools        |
| annovar         | bio     | 20191024                                       | ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others). | [http://annovar.openbioinformatics.org/en/latest/](http://annovar.openbioinformatics.org/en/latest/) | bio         |
| anserini        | -       | 0.9.4                                          | An open-source information retrieval toolkit built on Lucene that aims to bridge the gap between academic information retrieval and the practice of building real-world search applications. Usage: Binaries located in Anserini's target/appassembler/bin are on path. Anserini's folders bin, collections, docs, indexes, logs, runs, src, target, tools, as well as the files pom.xml and README.md are available at $EBROOTANSERINI. Eg. Indexing command for help: IndexCollection -h | [https://github.com/castorini/anserini](https://github.com/castorini/anserini)     |             |
| ansys           | phys    | 2019R3, 2020R2, 2021R1, 2021R2, 2022R1, 2022R2, 2023R1, 2023R2 | ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater.                                                | [http://www.ansys.com](http://www.ansys.com)                               | phys        |
| ansysedt        | -       | 2021R2, 2023R2                                  | ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater.                                                | [http://www.ansys.com](http://www.ansys.com)                               |             |
| ant             | tools   | 1.9.15, 1.10.8                                  | Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications. | [https://ant.apache.org/](https://ant.apache.org/)                           | tools        |
| antlr           | -       | 2.7.7                                          | ANTLR, ANother Tool for Language Recognition, (formerly PCCTS) is a language tool that provides a framework for constructing recognizers, compilers, and translators from grammatical descriptions containing Java, C#, C++, or Python actions. | [https://www.antlr2.org/](https://www.antlr2.org/)                           |             |
| apptainer       | -       | 1.1.3, 1.1.5, 1.1.6, 1.1.8, 1.2.4              | Apptainer/Singularity is an application containerization solution for High-Performance Computing (HPC). The goal of Apptainer is to allow for "mobility of computing": an application containerized on one Linux system should be able to run on another system, as it is, and without the need to reconcile software dependencies and Linux version differences between the source and target systems. | [https://apptainer.org](https://apptainer.org)                               |             |
| aragorn         | -       | 1.2.38                                         | a program to detect tRNA genes and tmRNA genes in nucleotide sequences                                                                                                                                                                | [http://mbio-serv2.mbioekol.lu.se/ARAGORN/](http://mbio-serv2.mbioekol.lu.se/ARAGORN/) |             |
| arrow           | tools   | 0.16.0, 10.0.1, 12.0.1, 13.0.0                 | Apache Arrow is a cross-language development platform for in-memory data.                                                                                                                                                           | [https://arrow.apache.org](https://arrow.apache.org)                           | tools        |
| ascp            | tools   | 3.5.4                                          | Aspera ASCP is a high speed file transfer software                                                                                                                                                                                | [http://downloads.asperasoft.com/](http://downloads.asperasoft.com/)             | tools        |
| autodock_vina   | chem    | 1.1.2                                          | AutoDock Vina is an open-source program for doing molecular docking.                                                                                                                                                                 | [http://vina.scripps.edu/index.html](http://vina.scripps.edu/index.html)       | chem        |
| bamtools        | bio     | 2.5.1                                          | BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.                                                                                                                                             | [https://github.com/pezmaster31/bamtools](https://github.com/pezmaster31/bamtools) | bio         |
| bbmap           | bio     | 38.86                                          | BBMap short read aligner, and other bioinformatic tools.                                                                                                                                                                         | [https://sourceforge.net/projects/bbmap/](https://sourceforge.net/projects/bbmap/) | bio         |
| bcftools        | bio     | 1.13                                           | Samtools is a suite of programs for interacting with high-throughput sequencing data. BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence variants                      | [https://www.htslib.org/](https://www.htslib.org/)                           | bio         |
| beagle          | -       | 5.4                                           | Beagle is a software package for phasing genotypes and for imputing ungenotyped markers.                                                                                                                                               | [https://faculty.washington.edu/browning/beagle/beagle.html](https://faculty.washington.edu/browning/beagle/beagle.html) |             |
| bedtools        | bio     | 2.30.0                                         | The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM.           | [https://github.com/arq5x/bedtools2](https://github.com/arq5x/bedtools2)       | bio         |
| beef            | chem    | 0.1.1                                          | BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at [http://dx.doi.org/10.1103/PhysRevB.85.235149](http://dx.doi.org/10.1103/PhysRevB.85.235149). | [http://suncat.stanford.edu/facility/software/functional](http://suncat.stanford.edu/facility/software/functional) | chem        |
| bio-searchio-hmmer | -       | 1.7.3                                          | Code to parse output from hmmsearch, hmmscan, phmmer and nhmmer, compatible with both version 2 and version 3 of the HMMER package from [http://hmmer.org](http://hmmer.org).                                                              | [https://metacpan.org/pod/Bio::SearchIO::hmmer3](https://metacpan.org/pod/Bio::SearchIO::hmmer3) |             |
| bioawk          | -       | 1.0                                           | Bioawk is an extension to Brian Kernighan's awk, adding the support of several common biological data formats, including optionally gzip'ed BED, GFF, SAM, VCF, FASTA/Q and TAB-delimited formats with column names.                   | [https://github.com/lh3/bioawk](https://github.com/lh3/bioawk)               |             |
| biobloomtools   | -       | 2.3.3                                          | BioBloom Tools (BBT) provides the means to create filters for a given reference and then to categorize sequences. This methodology is faster than alignment but does not provide mapping locations. BBT was initially intended to be used for pre-processing and QC applications like contamination detection, but is flexible to accommodate other purposes. This tool is intended to be a pipeline component to replace costly alignment steps. | [https://github.com/bcgsc/biobloom](https://github.com/bcgsc/biobloom)         |             |
| bioperl         | bio     | 1.7.7, 1.7.8                                  | Bioperl is the product of a community effort to produce Perl code which is useful in biology. Examples include Sequence objects, Alignment objects and database searching objects.                                                    | [https://bioperl.org/](https://bioperl.org/)                                 | bio         |
| blast           | bio     | 2.2.26                                         | Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences.                     | [http://blast.ncbi.nlm.nih.gov/](http://blast.ncbi.nlm.nih.gov/)               | bio         |
| blast+          | bio     | 2.12.0, 2.14.0                                 | Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences.                     | [https://blast.ncbi.nlm.nih.gov/](https://blast.ncbi.nlm.nih.gov/)               | bio         |
| blender         | vis     | 3.6.0                                          | Blender is the free and open source 3D creation suite. It supports the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering, compositing and motion tracking, even video editing and game creation.       | [https://www.blender.org/](https://www.blender.org/)                           | vis         |
| boost           | tools   | 1.72.0, 1.80.0                                  | Boost provides free peer-reviewed portable C++ source libraries.                                                                                                                                                                  | [https://www.boost.org/](https://www.boost.org/)                               | tools        |
| boost-mpi       | tools   | 1.72.0                                         | Boost provides free peer-reviewed portable C++ source libraries.                                                                                                                                                                  | [https://www.boost.org/](https://www.boost.org/)                               | tools        |
| bowtie2         | bio     | 2.4.1, 2.5.1                                  | Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes. | [http://bowtie-bio.sourceforge.net/bowtie2/index.shtml](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml) | bio         |
| bustools        | -       | 0.40.0                                         | bustools is a program for manipulating BUS files for single cell RNA-Seq datasets. It can be used to error correct barcodes, collapse UMIs, produce gene count or transcript compatibility count matrices, and is useful for many other tasks. See the kallisto bustools website for examples and instructions on how to use bustools as part of a single-cell RNA-seq workflow. | [https://github.com/BUStools/bustools](https://github.com/BUStools/bustools)       |             |
| bwa             | bio     | 0.7.17                                         | Burrows-Wheeler Aligner (BWA) is an efficient program that aligns relatively short nucleotide sequences against a long reference sequence such as the human genome.                                                                     | [http://bio-bwa.sourceforge.net/](http://bio-bwa.sourceforge.net/)             | bio         |
| cafe5           | -       | 5.1.0                                          | Software for Computational Analysis of gene Family Evolution The purpose of CAFE is to analyze changes in gene family size in a way that accounts for phylogenetic history and provides a statistical foundation for evolutionary inferences. The program uses a birth and death process to model gene gain and loss across a user-specified phylogenetic tree. The distribution of family sizes generated under this model can provide a basis for assessing the significance of the observed family size differences among taxa. | [https://github.com/hahnlab/CAFE5](https://github.com/hahnlab/CAFE5)           |             |
| canu            | bio     | 2.2                                           | Canu is a fork of the Celera Assembler, designed for high-noise single-molecule sequencing (such as the PacBio RS II or Oxford Nanopore MinION). Canu is a hierarchical assembly pipeline which runs in four steps: Detect overlaps in high-noise sequences using MHAP Generate corrected sequence consensus Trim corrected sequences Assemble trimmed corrected sequences | [https://github.com/marbl/canu](https://github.com/marbl/canu)               | bio         |
| cap3            | bio     | 20151002                                       | CAP3 assembly program                                                                                                                                                                                                              | [http://seq.cs.iastate.edu/](http://seq.cs.iastate.edu/)                     | bio         |
| capnproto       | -       | 0.7.0                                          | Cap'n Proto is an insanely fast data interchange format and capability-based RPC system.                                                                                                                                                 | [https://capnproto.org](https://capnproto.org)                               |             |
| catch2          | -       | 2.11.0                                         | A modern, C++-native, header-only, test framework for unit-tests, TDD and BDD - using C++11, C++14, C++17 and later (or C++03 on the Catch1.x branch)                                                                               | [https://github.com/catchorg/Catch2](https://github.com/catchorg/Catch2)       |             |
| ccsm            | geo     | 4_0_a02                                         | The Community Climate System Model (CCSM) is a coupled climate model for simulating the earth's climate system. Composed of four separate models simultaneously simulating the earth's atmosphere, ocean, land surface and sea-ice, and one central coupler component, the CCSM allows researchers to conduct fundamental research into the earth's past, present and future climate states. | [http://www.cesm.ucar.edu/models/ccsm4.0/](http://www.cesm.ucar.edu/models/ccsm4.0/) | geo         |
| cd-hit          | bio     | 4.8.1                                          | CD-HIT is a very widely used program for clustering and comparing protein or nucleotide sequences.                                                                                                                                          | [http://weizhongli-lab.org/cd-hit/](http://weizhongli-lab.org/cd-hit/)         | bio         |
| cdo             | geo     | 1.9.8, 1.9.10                                  | CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.                                                                                                                                      | [https://code.zmaw.de/projects/cdo](https://code.zmaw.de/projects/cdo)         | geo         |
| cellranger      | bio     | 2.1.0                                          | Cell Ranger is a set of analysis pipelines that process Chromium single cell 3-RNA-seq output to align reads, generate gene-cell matrices and perform clustering and gene expression analysis. Cell Ranger 1.2 and later support libraries generated by the Chromium Single Cell 3i-v1 and v2 reagent kits, whereas Cell Ranger 1.1 and earlier do not support v2 libraries. Cell Ranger includes four main gene expression pipelines: - cellranger mkfastq wraps Illumina's bcl2fastq to correctly demultiplex Chromium-prepared sequencing samples and to convert barcode and read data to FASTQ files. - cellranger count takes FASTQ files from cellranger mkfastq and performs alignment, filtering, and UMI counting. - cellranger aggr aggregates outputs from multiple runs of cellranger count, normalizing those runs to the same sequencing depth and then recomputing the gene-barcode matrices and analysis on the combined data. - cellranger reanalyze takes gene-barcode matrices produced by cellranger count or cellranger aggr and reruns the dimensionality reduction, clustering, and gene expression algorithms using tunable parameter settings. These pipelines combine Chromium-specific algorithms with the widely used RNA-seq aligner STAR. Output is delivered in standard BAM, MEX, CSV, HDF5 and HTML formats that are augmented with cellular information. | [https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome](https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome) | bio         |
| cereal          | -       | 1.3.0                                          | cereal is a header-only C++11 serialization library. cereal takes arbitrary data types and reversibly turns them into different representations, such as compact binary encodings, XML, or JSON. cereal was designed to be fast, light-weight, and easy to extend - it has no external dependencies and can be easily bundled with other code or used standalone. | [http://uscilab.github.io/cereal/](http://uscilab.github.io/cereal/)           |             |
| cesm            | geo     | 2.1.3                                          | The Community Earth System Model (CESM) is a fully-coupled, global climate model that provides state-of-the-art computer simulations of the Earth's past, present, and future climate states.                                         | [http://www.cesm.ucar.edu/models/cesm2/](http://www.cesm.ucar.edu/models/cesm2/) | geo         |
| cfitsio         | vis     | 3.48, 4.1.0                                   | CFITSIO is a library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format.                                                                                             | [https://heasarc.gsfc.nasa.gov/fitsio/](https://heasarc.gsfc.nasa.gov/fitsio/)   | vis         |
| cgal            | math    | 4.14.3, 5.5.2                                  | The goal of the CGAL Open Source Project is to provide easy access to efficient and reliable geometric algorithms in the form of a C++ library.                                                                                       | [https://www.cgal.org/](https://www.cgal.org/)                               | math        |
| cmake           | tools   | 3.18.4, 3.20.1, 3.21.4, 3.22.1, 3.23.1, 3.27.7 | CMake, the cross-platform, open-source build system. CMake is a family of tools designed to build, test and package software.                                                                                                         | [https://www.cmake.org](https://www.cmake.org)                               | tools        |
| code-server     | -       | 3.5.0, 3.12.0                                 | Run VS Code on any machine anywhere and access it in the browser.                                                                                                                                                                    | [https://github.com/cdr/code-server](https://github.com/cdr/code-server)       |             |
| coinmp          | -       | 1.8.4                                          | CoinMP is a C-API library that supports most of the functionality of CLP (Coin LP), CBC (Coin Branch-and-Cut), and CGL (Cut Generation Library) projects.                                                                               | [https://projects.coin-or.org/CoinMP](https://projects.coin-or.org/CoinMP)       |             |
| comsol          | phys    | 5.6, 6.0, 6.0.0.405, 6.1.0.357, 6.2            | COMSOL Multiphysics is a general-purpose software platform, based on advanced numerical methods, for modeling and simulating physics-based problems.                                                                                       | [https://www.comsol.com](https://www.comsol.com)                             | phys        |
| corset          | bio     | 1.09                                           | Corset is a command-line software program to go from a de novo transcriptome assembly to gene-level counts. Our software takes a set of reads that have been multi-mapped to the transcriptome (where multiple alignments per read were reported) and hierarchically clusters the transcripts based on the proportion of shared reads and expression patterns. It will report the clusters and gene-level counts for each sample, which are easily tested for differential expression with count based tools such as edgeR and DESeq. | [https://github.com/Oshlack/Corset/wiki](https://github.com/Oshlack/Corset/wiki) | bio         |
| cppzmq          | -       | 4.7.1                                          | cppzmq is a C++ binding for libzmq                                                                                                                                                                                                 | [https://github.com/zeromq/cppzmq](https://github.com/zeromq/cppzmq)           |             |
| cromwell        | tools   | 58                                            | Scientific workflow engine designed for simplicity and scalability. Trivially transition between one off use cases to massive scale production environments [http://cromwell.readthedocs.io/](http://cromwell.readthedocs.io/)                 | [https://cromwell.readthedocs.io/](https://cromwell.readthedocs.io/)           | tools        |
| cst             | phys    | 2020                                          | CST offers accurate, efficient computational solutions for electromagnetic design and analysis. CST simulation is user-friendly and enables users to choose the most appropriate method for the design and optimization of devices operating in a wide range of frequencies. Please note that we installed only Graphic frontend and command line interface. You need to provide your own license server in order to be able to run CST. | [https://www.cst.com/](https://www.cst.com/)                               | phys        |
| csvtk           | -       | 0.23.0                                         | Similar to FASTA/Q format in field of Bioinformatics, CSV/TSV formats are basic and ubiquitous file formats in both Bioinformatics and data science. People usually use spreadsheet software like MS Excel to process table data. However this is all by clicking and typing, which is not automated and is time-consuming to repeat, especially when you want to apply similar operations with different datasets or purposes. You can also accomplish some CSV/TSV manipulations using shell commands, but more code is needed to handle the header line. Shell commands do not support selecting columns with column names either. csvtk is convenient for rapid data investigation and also easy to integrate into analysis pipelines. It could save you lots of time in (not) writing Python/R scripts. | [https://github.com/shenwei356/csvtk](https://github.com/shenwei356/csvtk)       |             |
| cuda            | tools   | 10.2                                          | CUDA (formerly Compute Unified Device Architecture) is a parallel computing platform and programming model created by NVIDIA and implemented by the graphics processing units (GPUs) that they produce. CUDA gives developers access to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs. | [https://developer.nvidia.com/cuda-toolkit](https://developer.nvidia.com/cuda-toolkit) | tools        |
| cudnn           | math    | 8.0.3, 8.2.0, 8.6.0.163, 8.7.0.84, 8.9.5.29     | The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks.                                                                                                           | [https://developer.nvidia.com/cudnn](https://developer.nvidia.com/cudnn)         | math        |
| cufflinks       | bio     | 2.2.1                                          | Transcript assembly, differential expression, and differential regulation for RNA-Seq                                                                                                                                                 | [http://cole-trapnell-lab.github.io/cufflinks/](http://cole-trapnell-lab.github.io/cufflinks/) | bio         |
| cusparselt      | -       | 0.4.0.7                                         | NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general matrix-matrix operations in which at least one operand is a sparse matrix                                                                                       | [https://docs.nvidia.com/cuda/cusparselt/index.html](https://docs.nvidia.com/cuda/cusparselt/index.html) |             |
| custom-ctypes   | -       | 1.1                                           | This repository is to try to fix discovery of libraries with Python's ctypes module. It changes the behavior of find_library to return absolute paths to shared objects rather than just the names.                                   | [https://github.com/ComputeCanada/custom_ctypes/](https://github.com/ComputeCanada/custom_ctypes/) |             |
| cutensor        | -       | 1.5.0.3, 1.7.0.1                              | cuTENSOR is a high-performance CUDA library for tensor primitives.                                                                                                                                                                 | [https://developer.nvidia.com/cutensor](https://developer.nvidia.com/cutensor)     |             |
| cvit            | -       | 1.2.1                                          | A genomic linkage feature visualization tools based on Perl.                                                                                                                                                                       | [https://sourceforge.net/projects/cvit/](https://sourceforge.net/projects/cvit/)     |             |
| ddt-cpu         | tools   | 20.2, 22.0.1                                  | Profiler and debugger, combining Arm MAP and Arm DDT.                                                                                                                                                                                | [http://www.allinea.com](http://www.allinea.com)                               | tools        |
| ddt-gpu         | tools   | 20.2, 22.0.1                                  | Profiler and debugger, combining Arm MAP and Arm DDT, with support for CUDA.                                                                                                                                                          | [http://www.allinea.com](http://www.allinea.com)                               | tools        |
| delly           | -       | 0.8.5                                          | Delly is an integrated structural variant (SV) prediction method that can discover, genotype and visualize deletions, tandem duplications, inversions and translocations at single-nucleotide resolution in short-read massively parallel sequencing data. | [https://github.com/dellytools/delly](https://github.com/dellytools/delly)       |             |
| diamond         | bio     | 2.1.6                                          | Accelerated BLAST compatible local sequence aligner                                                                                                                                                                               | [https://github.com/bbuchfink/diamond](https://github.com/bbuchfink/diamond)     | bio         |
| dl_monte        | -       | 2.07                                          | General-purpose Monte Carlo molecular simulation                                                                                                                                                                                   | [https://gitlab.com/dl_monte](https://gitlab.com/dl_monte)                   |             |
| dorado          | -       | 0.2.1, 0.2.2, 0.3.0                           | Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.                                                                                                                                          | [https://github.com/nanoporetech/dorado](https://github.com/nanoporetech/dorado) |             |
| dotnet-core     | -       | 3.1.8, 5.0.12, 6.0.0                           | .NET is a free, cross-platform, open source developer platform for building many different types of applications. With .NET, you can use multiple languages, editors, and libraries to build for web, mobile, desktop, gaming, and IoT. Contains the SDK and the Runtime. | [https://www.microsoft.com/net/](https://www.microsoft.com/net/)             |             |
| double-conversion | -       | 3.1.5                                          | Efficient binary-decimal and decimal-binary conversion routines for IEEE doubles.                                                                                                                                                       | [https://github.com/google/double-conversion](https://github.com/google/double-conversion) |             |
| dssp            | chem    | 3.1.4                                          | The DSSP algorithm is the standard method for assigning secondary structure to the amino acids of a protein, given the atomic-resolution coordinates of the protein. If you use DSSP, please quote these two articles: 1) A series of PDB related databases for everyday needs. Wouter G Touw, Coos Baakman, Jon Black, Tim AH te Beek, E Krieger, Robbie P Joosten, Gert Vriend. Nucleic Acids Research 2015 January; 43(Database issue): D364-D368. 2) Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features. Kabsch W, Sander C, Biopolymers. 1983 22 2577-2637. PMID: 6667333; UI: 84128824. | [https://swift.cmbi.umcn.nl/gv/dssp/](https://swift.cmbi.umcn.nl/gv/dssp/)     | chem        |
| eccodes         | geo     | 2.15.0, 2.19.0, 2.21.0, 2.22.1                 | ecCodes is a package developed by ECMWF which provides an application programming interface and a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2, WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding). | [https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home](https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home) | geo         |
| eigen           | math    | 3.3.7                                          | Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms.                                                                                                                      | [https://eigen.tuxfamily.org/index.php?title=Main_Page](https://eigen.tuxfamily.org/index.php?title=Main_Page) | math        |
| elpa            | math    | 2020.05.001                                    | Eigenvalue SoLvers for Petaflop-Applications                                                                                                                                                                                    | [https://elpa.rzg.mpg.de](https://elpa.rzg.mpg.de)                           | math        |
| emboss          | bio     | 6.6.0                                          | EMBOSS is 'The European Molecular Biology Open Software Suite' . EMBOSS is a free Open Source software analysis package specially developed for the needs of the molecular biology (e.g. EMBnet) user community.                           | [https://emboss.sourceforge.net/](https://emboss.sourceforge.net/)             | bio         |
| energyplus       | -       | 9.3.0, 23.2.0                                 | EnergyPlus is a whole building energy simulation program that engineers, architects, and researchers use to model both energy consumptionfor heating, cooling, ventilation, lighting and plug and process loadsand water use in buildings. | [https://energyplus.net/downloads](https://energyplus.net/downloads)           |             |
| expat           | tools   | 2.2.9, 2.2.10, 2.4.8                           | Expat is an XML parser library written in C. It is a stream-oriented parser in which an application registers handlers for things the parser might find in the XML document (like start tags)                                       | [https://libexpat.github.io](https://libexpat.github.io)                     | tools        |
| falcon          | bio     | 1.8.8                                          | Falcon: a set of tools for fast aligning long reads for consensus and assembly                                                                                                                                                     | [https://github.com/PacificBiosciences/FALCON](https://github.com/PacificBiosciences/FALCON) | bio         |
| fastahack       | -       | 1.0.0                                          | Utilities for indexing and sequence extraction from FASTA files.                                                                                                                                                                    | [https://github