# Modules avx2

This document lists software modules available with AVX2 support.  The information includes module name, type, versions, description, homepage URL, and relevant keywords.


| Module             | Type    | Versions                                      | Description                                                                                                                                                                                               | URL                                                                                             | Keyword(s) |
|----------------------|---------|-------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------|-------------|
| 4ti2                | -       | 1.6.9                                         | A software package for algebraic, geometric and combinatorial problems on linear spaces.                                                                                                                            | https://4ti2.github.io/                                                                     |             |
| abaqus              | phys    | 2021                                          | Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA.                                                                                             | http://www.simulia.com/products/abaqus_fea.html                                              | phys        |
| abinit              | chem    | 9.2.2, 9.6.2, 10.0.3, 10.2.3                   | ABINIT is a package whose main program allows one to find the total energy, charge density and electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional Theory (DFT), using pseudopotentials and a planewave or wavelet basis. | https://www.abinit.org/                                                                     | chem        |
| abricate            | -       | 1.0.0                                         | Mass screening of contigs for antimicrobial and virulence genes.                                                                                                                                                 | https://github.com/tseemann/abricate                                                          |             |
| abseil              | -       | 20230125.3                                     | Abseil is an open-source collection of C++ library code designed to augment the C++ standard library. The Abseil library code is collected from Google's own C++ code base, has been extensively tested and used in production, and is the same code we depend on in our daily coding lives. | https://abseil.io/                                                                         |             |
| abyss               | bio     | 2.1.5, 2.2.5, 2.3.7                           | Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler.                                                                                                                            | https://www.bcgsc.ca/platform/bioinfo/software/abyss                                         | bio         |
| actc                | -       | 1.1                                          | ACTC converts independent triangles into triangle strips or fans.                                                                                                                                                  | https://sourceforge.net/projects/actc                                                          |             |
| admixture           | bio     | 1.3.0                                         | ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much more rapidly using a fast numerical optimization algorithm. | https://dalexander.github.io/admixture/                                                        | bio         |
| adol-c              | -       | 2.7.2                                         | The package ADOL-C (Automatic Differentiation by OverLoading in C++) facilitates the evaluation of first and higher derivatives of vector functions that are defined by computer programs written in C or C++. The resulting derivative evaluation routines may be called from C/C++, Fortran, or any other language that can be linked with C. | https://projects.coin-or.org/ADOL-C                                                          |             |
| adtree              | -       | 1.1.2                                         | AdTree: Accurate, Detailed, and Automatic Modelling of Laser-Scanned Trees.                                                                                                                                    | https://github.com/tudelft3d/AdTree                                                          |             |
| advisor             | tools   | 2020.3                                        | Vectorization Optimization and Thread Prototyping - Vectorize and thread code or performance dies - Easy workflow + data + tips = faster code faster - Prioritize, Prototype and Predict performance gain. | https://software.intel.com/intel-advisor-xe                                                | tools        |
| afni                | bio     | 20.3.05, 21.2.10, 22.1.12, 23.1.00, 23.1.08, 24.1.03 | AFNI is a set of C programs for processing, analyzing, and displaying functional MRI (FMRI) data - a technique for mapping human brain activity.                                                              | http://afni.nimh.nih.gov/                                                                     | bio         |
| alevin-fry          | -       | 0.8.2                                         | alevin-fry is an efficient and flexible tool for processing single-cell sequencing data, currently focused on single-cell transcriptomics and feature barcoding.                                                 | https://github.com/COMBINE-lab/alevin-fry                                                   |             |
| alm                 | -       | 2.0.0_dev.2                                    | Spglib is a software for calculating harmonic and anharmonic interatomic force constants in solids and molecules.                                                                                                    | https://github.com/ttadano/ALM                                                               |             |
| almosthere          | -       | 1.0.10, 1.0.15                                 | Progress indicator C library. ATHR is a simple yet powerful progress indicator library that works on Windows, Linux, and macOS. It is non-blocking as the progress update is done via a dedicated, lightweight thread, as to not impair the performance of the calling program. | https://github.com/horta/almosthere                                                          |             |
| alpscore            | phys    | 2.2.0                                         | The ALPSCore project, based on the ALPS (Algorithms and Libraries for Physics Simulations) project, provides generic algorithms and utilities for physics problems. It strives to increase software reuse in the physics community. For copyright see COPYRIGHT.TXT For licensing see LICENSE.TXT For acknowledgment in scientific publications see ACKNOWLEDGE.TXT | https://github.com/ALPSCore/ALPSCore                                                        | phys        |
| amber               | chem    | 22.5-23.5, 18.14-18.17, 20.9-20.15, 20.12-20.15 | Amber (originally Assisted Model Building with Energy Refinement) is software for performing molecular dynamics and structure prediction.                                                                          | http://ambermd.org/amber.html                                                                 | chem        |
| ambertools          | chem    | 20, 21, 22, 23.5                              | AmberTools consists of several independently developed packages that work well by themselves, and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations, with either explicit water or generalized Born solvent models. | https://ambermd.org/AmberTools.php                                                             | chem        |
| amos                | -       | 3.1.0                                          | The AMOS consortium is committed to the development of open-source whole genome assembly software.                                                                                                                  | http://amos.sourceforge.net                                                                   |             |
| ampl-mp             | -       | 3.1.0                                          | An open-source library for mathematical programming.                                                                                                                                                       | https://github.com/ampl/mp                                                                   |             |
| amrfinderplus       | -       | 3.11.18, 3.11.26, 3.12.8                       | This software and the accompanying database identify acquired antimicrobial resistance genes in bacterial protein and/or assembled nucleotide sequences as well as known resistance-associated point mutations for several taxa. With AMRFinderPlus we added select members of additional classes of genes such as virulence factors, biocide, heat, acid, and metal resistance genes. | https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus                                      |             |
| amrplusplus         | -       | 2.0-20200114                                  | AmrPlusPlus v2.0 can process the raw data from the sequencer, identify the fragments of DNA, and count them. It also provides a count of the polymorphisms that occur in each DNA fragment with respect to the reference database. | https://megares.meglab.org/amrplusplus/latest/html/v2                                       |             |
| andi                | -       | 0.14                                          | This is the andi program for estimating the evolutionary distance between closely related genomes. These distances can be used to rapidly infer phylogenies for big sets of genomes. Because andi does not compute full alignments, it is so efficient that it scales even up to thousands of bacterial genomes. | https://github.com/evolbioinf/andi/                                                          |             |
| angsd               | bio     | 0.933, 0.935, 0.936, 0.939, 0.940              | Program for analysing NGS data.                                                                                                                                                                            | http://www.popgen.dk/angsd                                                                    | bio         |
| annovar             | bio     | 20191024                                       | ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others). | http://annovar.openbioinformatics.org/en/latest/                                             | bio         |
| anserini            | -       | 0.9.4                                         | An open-source information retrieval toolkit built on Lucene that aims to bridge the gap between academic information retrieval and the practice of building real-world search applications. Usage: Binaries located in Anserini's target/appassembler/bin are on path. Anserini's folders bin, collections, docs, indexes, logs, runs, src, target, tools, as well as the files pom.xml and README.md are available at $EBROOTANSERINI. Eg. Indexing command for help: IndexCollection -h | https://github.com/castorini/anserini                                                        |             |
| ansys               | phys    | 2024R2.04, 2024R1.03, 2025R1.02, 2019R3, 2020R2, 2021R1, 2021R2, 2022R1, 2022R2, 2023R1, 2023R2, 2024R1, 2025R1 | ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater.                     | http://www.ansys.com                                                                       | phys        |
| ansysedt            | -       | 2021R2, 2023R2                                 | ANSYS simulation software enables organizations to confidently predict how their products will operate in the real world. We believe that every product is a promise of something greater.                     | http://www.ansys.com                                                                       |             |
| ant                 | tools   | 1.9.15, 1.10.8, 1.10.14                        | Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications. | https://ant.apache.org/                                                                     | tools        |
| antlr               | -       | 2.7.7                                         | ANTLR, ANother Tool for Language Recognition, (formerly PCCTS) is a language tool that provides a framework for constructing recognizers, compilers, and translators from grammatical descriptions containing Java, C#, C++, or Python actions. | https://www.antlr2.org/                                                                     |             |
| ants                | vis     | 2.3.2, 2.3.5, 2.4.4, 2.5.0                     | ANTs extracts information from complex datasets that include imaging. ANTs is useful for managing, interpreting and visualizing multidimensional data.                                                       | https://stnava.github.io/ANTs/                                                              | vis         |
| any2fasta           | -       | 0.4.2                                         | Convert various sequence formats to FASTA.                                                                                                                                                                | https://github.com/tseemann/any2fasta                                                          |             |
| aocl-blas           | -       | 5.1                                           | AOCL-BLAS is AMD's optimized version of BLAS targeted for AMD EPYC and Ryzen CPUs.                                                                                                                            | https://github.com/amd/blis                                                                  |             |
| aocl-lapack         | -       | 5.1                                           | AOCL-LAPACK is AMD's optimized version of LAPACK targeted for AMD EPYC and Ryzen CPUs.                                                                                                                           | https://github.com/amd/libflame                                                                |             |
| apbs                | chem    | 1.3                                           | APBS is a software package for modeling biomolecular solvation through solution of the Poisson-Boltzmann equation (PBE), one of the most popular continuum models for describing electrostatic interactions between molecular solutes in salty, aqueous media. | http://www.poissonboltzmann.org/apbs                                                          | chem        |
| apptainer           | -       | 1.1.3, 1.1.5, 1.1.6, 1.1.8, 1.2.4, 1.3.4, 1.3.5 | Apptainer/Singularity is an application containerization solution for High-Performance Computing (HPC). The goal of Apptainer is to allow for "mobility of computing": an application containerized on one Linux system should be able to run on another system, as it is, and without the need to reconcile software dependencies and Linux version differences between the source and target systems. | https://apptainer.org                                                                       |             |
| aragorn             | -       | 1.2.38, 1.2.41                               | a program to detect tRNA genes and tmRNA genes in nucleotide sequences.                                                                                                                                          | http://www.ansikte.se/ARAGORN/                                                                |             |
| arb                 | -       | 2.19.0                                        | Arb is a C library for arbitrary-precision interval arithmetic. It has full support for both real and complex numbers. The library is thread-safe, portable, and extensively tested.                     | https://arblib.org/                                                                         |             |
| arcs                | bio     | 1.2.1, 1.2.5, 1.2.7                           | Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data.                                                                                                                            | https://github.com/bcgsc/arcs                                                                | bio         |
| argtable            | tools   | 2.13                                          | Argtable is an ANSI C library for parsing GNU style command line options with a minimum of fuss.                                                                                                                      | http://argtable.sourceforge.net/                                                              | tools        |
| arioc               | -       | 1.43                                          | ACTC converts independent triangles into triangle strips or fans.                                                                                                                                                  | https://sourceforge.net/projects/actc                                                          |             |
| arks                | -       | 1.0.4                                         | Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data.                                                                                                                            | https://github.com/bcgsc/arcs                                                                |             |
| armadillo           | math    | 9.900.2, 12.6.4                               | Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards a good balance between speed and ease of use. Integer, floating point and complex numbers are supported, as well as a subset of trigonometric and statistics functions. | https://arma.sourceforge.net/                                                              | math        |
| arpack-ng           | math    | 3.7.0, 3.8.0, 3.9.0, 3.9.1                     | ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.                                                                                                                      | https://github.com/opencollab/arpack-ng                                                      | math        |
| arrayfire           | -       | 3.7.3, 3.8.2, 3.9.0                           | ArrayFire is a general-purpose library that simplifies the process of developing software that targets parallel and massively-parallel architectures including CPUs, GPUs, and other hardware acceleration devices. | https://arrayfire.com/                                                                     |             |
| arrow               | tools   | 0.16.0, 0.17.1, 2.0.0, 5.0.0, 8.0.0, 9.0.0, 10.0.1, 11.0.0, 12.0.1, 13.0.0, 14.0.0, 14.0.1, 15.0.1, 16.1.0, 17.0.0, 18.1.0, 19.0.1 | Apache Arrow is a cross-language development platform for in-memory data.                                                                                                                                  | https://arrow.apache.org                                                                     | tools        |
| ascp                | tools   | 3.5.4                                          | Aspera ASCP is a high speed file transfer software.                                                                                                                                                          | http://downloads.asperasoft.com/                                                              | tools        |
| aspect              | -       | 2.3.0, 2.4.0                                 | ASPECT: Advanced Solver for Problems in Earth's ConvecTion. An extensible code written in C++ to support research in simulating convection in the Earth's mantle and elsewhere to provide the geosciences with a well-documented and extensible code base for their research needs and to create an open, inclusive, participatory community providing users and developers with a state-of-the-art, comprehensive software that performs well while being simple to extend. | https://aspect.geodynamics.org/                                                              |             |
| assimp              | -       | 5.0.0, 5.2.5                                 | Open Asset Import Library (assimp) is a library to import and export various 3d-model-formats including scene-post-processing to generate missing render data.                                                 | https://github.com/assimp/assimp                                                              |             |
| astrid              | -       | 2.2.1                                         | ASTRID-2 is a method for estimating species trees from gene trees.                                                                                                                                                | https://github.com/pranjalv123/ASTRID                                                        |             |
| atat                | -       | 3.36                                          | ATAT is a generic name that refers to a collection of alloy theory tools developed by Axel van de Walle, in collaboration with various research groups and with various sources of financial support.       | https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/                                  |             |
| atom                | chem    | 4.2.7_2                                       | ATOM is the name of a program originally written (circa 1982) by Sverre Froyen at the University of California at Berkeley, modified starting in 1990 by Norman Troullier and Jose Luis Martins at the University of Minnesota, and currently maintained by Alberto Garcia, who added some features and made substantial structural changes to the April 1990 (5.0) Minnesota version while at Berkeley and elsewhere. | https://departments.icmab.es/leem/siesta/Pseudopotentials/Code/downloads.html                 | chem        |
| atomicrex           | chem    | 1.0.20181114                                  | atomicrex is a versatile tool for the construction of advanced atomistic models. It is written in C++ and Python. It was primarily developed to fit interatomic potential models. Thanks to its flexible generic structure its application range, however, is much larger. In a general sense, it allows one to develop models that describe a given property as a function of an atomic (or atom-like) configuration. The property in question can be scalar or vectorial in nature, and could represent e.g., total energies and forces, or eventually electronic eigen energies. It thus provides already the basic framework for constructing in the future for example tight binding models. | http://atomicrex.gitlab.io/atomicrex/                                                        | chem        |
| atomphaw            | chem    | 4.1.0.6, 4.2.0.3                              | AtomPAW is a Projector-Augmented Wave Dataset Generator that can be used both as a standalone program and a library.                                                                                             | http://users.wfu.edu/natalie/papers/pwpaw/man.html                                           | chem        |
| augustus            | bio     | 3.3.3, 3.4.0, 3.5.0                           | AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.                                                                                                                                       | https://bioinf.uni-greifswald.de/augustus/                                                   | bio         |
| autodiff            | -       | 0.6.0, 0.6.12                                 | autodiff is a C++17 library that uses modern and advanced programming techniques to enable automatic computation of derivatives in an efficient, easy, and intuitive way.                                         | https://github.com/autodiff/autodiff                                                          |             |
| autodock-gpu        | -       | 1.5.3, 1.6                                     | OpenCL and Cuda accelerated version of AutoDock. It leverages its embarrassingly parallelizable LGA by processing ligand-receptor poses in parallel over multiple compute units. AutoDock is a suite of automated docking tools. It is designed to predict how small molecules, such as substrates or drug candidates, bind to a receptor of known 3D structure. | https://github.com/ccsb-scripps/AutoDock-GPU                                                |             |
| autodock-vina       | -       | 1.2.6                                         | AutoDock Vina is an open-source program for doing molecular docking.                                                                                                                                             | https://vina.scripps.edu/                                                                     |             |
| autodock_vina       | chem    | 1.1.2                                         | AutoDock Vina is an open-source program for doing molecular docking.                                                                                                                                             | http://vina.scripps.edu/index.html                                                             | chem        |
| bacio               | -       | 2.6.0                                          | This library performs binary I/O for the NCEP models, processing unformatted byte-addressable data records, and transforming the little endian files and big endian files. This is part of the NCEPLIBS project. | https://github.com/NOAA-EMC/NCEPLIBS-bacio/                                                  |             |
| bamm                | bio     | 2.5.0                                         | BAMM is oriented entirely towards detecting and quantifying heterogeneity in evolutionary rates. It uses reversible jump Markov chain Monte Carlo to automatically explore a vast universe of candidate models of lineage diversification and trait evolution. | http://bamm-project.org/                                                                     | bio         |
| bamtools            | bio     | 2.5.1, 2.5.2                                 | BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files.                                                                                                                      | https://github.com/pezmaster31/bamtools                                                       | bio         |
| bamutil             | bio     | 1.0.14                                        | BamUtil is a repository that contains several programs that perform operations on SAM/BAM files. All of these programs are built into a single executable, bam.                                            | http://genome.sph.umich.edu/wiki/BamUtil                                                      | bio         |
| barracuda           | -       | 0.7.107h                                      | Barracuda is a high-speed sequence aligner based on Sanger's BWA and utilizes the latest Nvidia CUDA architecture for accelerating alignments of sequence reads generated by next-generation sequencers. | https://sourceforge.net/projects/seqbarracuda/                                               |             |
| barrnap             | -       | 0.9                                           | Barrnap predicts the location of ribosomal RNA genes in genomes. It supports bacteria (5S,23S,16S), archaea (5S,5.8S,23S,16S), metazoan mitochondria (12S,16S) and eukaryotes (5S,5.8S,28S,18S). It takes FASTA DNA sequence as input, and write GFF3 as output. It uses the new nhmmer tool that comes with HMMER 3.1 for HMM searching in RNA:DNA style. Multithreading is supported and one can expect roughly linear speed-ups with more CPUs. | https://github.com/tseemann/barrnap                                                          |             |
| bayesass            | bio     | 3.0.4                                         | BayesAss: Bayesian Inference of Recent Migration Using Multilocus Genotypes.                                                                                                                                  | http://www.rannala.org/?page_id=245                                                            | bio         |
| bayesass3-snps      | -       | 1.1                                          | Modification of BayesAss 3.0.4 to allow handling of large SNP datasets generated via methods such as RADseq protocols. This is an enhancement of the code produced by the Rannala Research Group: http://www.rannala.org/inference-of-recent-migration Most changes will not be obvious to the average user. However, the most important changes will now allow the user to input large SNP datasets that previously would have caused BayesAss to crash with a "segmentation fault" error. The upper limit of 420 loci has been removed. | https://github.com/stevemussmann/BayesAss3-SNPs                                             |             |
| bayescan            | bio     | 2.1                                           | BayeScan aims at identifying candidate loci under natural selection from genetic data, using differences in allele frequencies between populations.                                                                 | http://cmpg.unibe.ch/software/BayeScan/                                                       | bio         |
| baypass             | bio     | 2.1, 2.2                                      | The package BayPass is a population genomics software which is primarily aimed at identifying genetic markers subjected to selection and/or associated to population-specific covariates (e.g., environmental variables, quantitative or categorical phenotypic characteristics). | http://www1.montpellier.inra.fr/CBGP/software/baypass/                                       | bio         |
| bazel               | tools   | 3.6.0                                          | Bazel is a build tool that builds code quickly and reliably. It is used to build the majority of Google's software.                                                                                             | https://bazel.io/                                                                           | tools        |
| bbmap               | bio     | 37.78, 38.86, 39.06                           | BBMap short read aligner, and other bioinformatic tools.                                                                                                                                                  | https://sourceforge.net/projects/bbmap/                                                       | bio         |
| bcftools            | bio     | 1.9, 1.10.2, 1.11, 1.13, 1.16, 1.18, 1.19          | Samtools is a suite of programs for interacting with high-throughput sequencing data. BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence variants. | https://www.htslib.org/                                                                     | bio         |
| bcl-convert          | -       | 4.2.4, 4.2.7                                 | The Illumina BCL Convert v4.0 is a standalone local software app that converts the Binary Base Call (BCL) files produced by Illumina sequencing systems to FASTQ files.                                      | https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html                   |             |
| bcl2fastq2          | bio     | 2.20.0                                        | bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by Illumina sequencing systems to standard FASTQ file formats for downstream analysis.                                  | https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html | bio         |
| beagle              | -       | 5.4, 5.4-240301                              | Beagle is a software package for phasing genotypes and for imputing ungenotyped markers.                                                                                                                      | https://faculty.washington.edu/browning/beagle/beagle.html                                     |             |
| beagle-lib          | bio     | 3.1.2, 4.0.0, 4.0.1                           | beagle-lib is a high-performance library that can perform the core calculations at the heart of most Bayesian and Maximum Likelihood phylogenetics packages.                                                   | https://github.com/beagle-dev/beagle-lib                                                      | bio         |
| beast               | bio     | 2.6.3, 2.7.5                                 | BEAST is a cross-platform program for Bayesian MCMC analysis of molecular sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies but is also a framework for testing evolutionary hypotheses without conditioning on a single tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted proportional to its posterior probability. | http://beast2.org                                                                          | bio         |
| bedops              | -       | 2.4.39, 2.4.41                               | BEDOPS is an open-source command-line toolkit that performs highly efficient and scalable Boolean and other set operations, statistical calculations, archiving, conversion and other management of genomic data of arbitrary scale. Tasks can be easily split by chromosome for distributing whole-genome analyses across a computational cluster. | http://bedops.readthedocs.io/en/latest/index.html                                           |             |
| bedtools            | bio     | 2.29.2, 2.30.0, 2.31.0                        | The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM. | https://github.com/arq5x/bedtools2                                                           | bio         |
| beef                | chem    | 0.1.1                                          | BEEF is a library-based implementation of the Bayesian Error Estimation Functional, suitable for linking against by Fortran- or C-based DFT codes. A description of BEEF can be found at http://dx.doi.org/10.1103/PhysRevB.85.235149. | http://suncat.stanford.edu/facility/software/functional                                      | chem        |
| berkeleygw          | phys    | 2.1.0, 3.0.1, 4.0                             | The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle properties and the optical responses of a large variety of materials from bulk periodic crystals to nanostructures such as slabs, wires and molecules. We ask that you please cite the following papers when using results obtained from the BerkeleyGW package: - Mark S. Hybertsen and Steven G. Louie, Electron correlation in semiconductors and insulators: Band gaps and quasiparticle energies, Phys. Rev. B 34, 5390 (1986). - Michael Rohlfing and Steven G. Louie, Electron-hole excitations and optical spectra from first principles, Phys. Rev. B 62, 4927 (2000). - Jack Deslippe, Georgy Samsonidze, David A. Strubbe, Manish Jain, Marvin L. Cohen, and Steven G. Louie, BerkeleyGW: A Massively Parallel Computer Package for the Calculation of the Quasiparticle and Optical Properties of Materials and Nanostructures, Comput. Phys. Commun. 183, 1269 (2012) (https://arxiv.org/abs/1111.4429). | http://www.berkeleygw.org                                                                   | phys        |
| bgcdist             | -       | 1.03                                          | bgc implements Bayesian estimation of genomic clines to quantify introgression at many loci. Models are available for linked loci, genotype uncertainty and sequence errors. Models for genotype uncertainty and sequence error can be used to estimate genomic cline parameters from next-generation sequence data. | https://sites.google.com/site/bgcsoftware/                                                  |             |
| bgen-lib            | -       | 1.1.7                                         | This repository contains a reference implementation of the BGEN format, written in C++. The library can be used as the basis for BGEN support in other software, or as a reference for developers writing their own implementations of the BGEN format. What's included? This repository contains the library itself, a set of example data files, and a number of example programs (e.g. bgen_to_vcf) that demonstrate the use of the library API. In addition, a number of utilities built using the library are also included in this repository: bgenix - a tool to index and efficiently retrieve subsets of a BGEN file. cat-bgen - a tool to efficiently concatenate BGEN files. edit-bgen - a tool to edit BGEN file metadata. An R package called rbgen is also constructed in the build directory. See the rbgen wiki page for more information on using this package. | https://enkre.net/cgi-bin/code/bgen                                                           |             |
| bifrost             | -       | 1.0.5                                         | Parallel construction, indexing and querying of colored and compacted de Bruijn graphs.                                                                                                                            | https://github.com/pmelsted/bifrost                                                          |             |
| bigdft              | chem    | 1.8.3, 1.9.5                                 | BigDFT is a DFT massively parallel electronic structure code using a wavelet basis set with the capability to use a linear scaling method. Wavelets form a real space basis set distributed on an adaptive mesh (two levels of resolution in our implementation). GTH or HGH pseudopotentials are used to remove the core electrons. The code BigDFT is available in ABINIT v5.5 and higher but can also be downloaded in a standalone version from the website. Thanks to our Poisson solver based on a Green function formalism, periodic systems, surfaces and isolated systems can be simulated with explicit boundary conditions. The Poisson solver can also be downloaded and used independently and is integrated in ABINIT, Octopus and CP2K. The code is free software, available under GNU-GPL license and the BigDFT developer community encourages anyone willing to contribute to join the team. | http://bigdft.org/Wiki/index.php?title=BigDFT_website                                        | chem        |
| bio-searchio-hmmer  | -       | 1.7.3                                         | Code to parse output from hmmsearch, hmmscan, phmmer and nhmmer, compatible with both version 2 and version 3 of the HMMER package from http://hmmer.org.                                                       | https://metacpan.org/pod/Bio::SearchIO::hmmer3                                              |             |
| bioawk              | -       | 1.0                                           | Bioawk is an extension to Brian Kernighan's awk, adding the support of several common biological data formats,